sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -464,13 +464,21 @@ class EthService(
   def traceTransaction(transactionHash: String, traceParams: TraceParams): Object = {
     val requestedTransaction = getTransactionAndReceipt(transactionHash) { (tx, receipt) =>
       new EthereumTransactionView(receipt, tx)
-    }.orNull
+    } match {
+      case Some(tx) => tx
+      case None =>
+        return ApiResponseUtil.toResponse(
```

Comment:
We need to throw an `RpcException` here, any "normal" return value would get sent back as a success result.